heartbeat/docs/getting-started.asciidoc

```diff
@@ -0,0 +1,261 @@
+[[heartbeat-getting-started]]
+== Getting Started With Heartbeat
+
+To get started with your own Heartbeat setup, install and configure these
+related products:
+
+ * Elasticsearch for storage and indexing the data.
+ * Kibana for the UI.
+ * Logstash (optional) for inserting data into Elasticsearch.
+ 
+See {libbeat}/getting-started.html[Getting Started with Beats and the Elastic Stack]
+for more information.
+
+After installing the Elastic Stack, read the following topics to learn how to
+install, configure, and run Heartbeat:
+
+* <<heartbeat-installation>>
+* <<heartbeat-configuration>>
+* <<heartbeat-template>>
+* <<heartbeat-starting>>
+* <<heartbeat-sample-dashboards>>
+* <<heartbeat-command-line>>
+* <<directory-layout>>
+
+
+[[heartbeat-installation]]
+=== Step 1: Installing Heartbeat
+
+Unlike most Beats, which you install on edge nodes, you typically install
+Heartbeat as part of monitoring service that runs on a separate machine
+and possibly even outside of the network where the services that you want to
+monitor are running. 
+
+//Steffen: I wonder if we need a topic that explores deployment scenarios in more detail (like installing on a sub-network where there's a filewall etc. I don't want to complicate the getting started with this level of details. WDYT?
```

Comment:
Ok, I've added it to my list of future Beats enhancements just so I don't forget to add it later.